.cirrus.yml

```diff
@@ -36,10 +36,7 @@ task:
       compile_host_script: |
         cd $ENGINE_PATH/src
         ./flutter/tools/gn --unoptimized --full-dart-sdk
-        ninja -C out/host_debug_unopt
-      test_host_script: |
-        cd $ENGINE_PATH/src
-        ./flutter/testing/run_tests.sh host_debug_unopt
+        ninja -C out/host_debug_unopt -j 20
```

Comment:
Ninja automatically selects the right job level. I don't think selecting parallel jobs would matter. If we wanted to make it so that ninja did not start jobs based on a specified load average, there is the `-l` flag which specifically says:

```
-l N     do not start new jobs if the load average is greater than N
```

Reply:
Yes, framework tests are the major factor, but removing the C++ unit tests takes about 10 minutes off the execution time, making it less likely that the framework tests will time out.

Reply:
Removed the `-j 20` flag.